### DISCUSSION TITLE
Add a React-compatible datetime picker to the UI library

### DISCUSSION THREAD

**Discussion by ethirolle** [grafana/grafana#38319] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Add a datetime picker to the UI library, so that plugin developers can include this component in new plugins based on React js library.

**Why is this needed**:
To allow adding a datetime picker to custom plugins.

Per customer comment:

> I have a custom panel plugin and I need to add a time picker in the options menu. In the Angular framework I was able to add an input field of type datetime-local in the options html. However in the react framework I don't see any equivalent to this. 

Screenshot of Angular-based datetime picker, which might serve as inspiration for what is needed:
![image](https://user-images.githubusercontent.com/10763196/98572516-b9e84800-2283-11eb-8311-33672e81eb2a.png)

**Comment by zoltanbedi** [comment]
This looks like a simple https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local you can use the `Input` component from Grafana ui with a type `datetime-local`.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
